Source/WebCore/platform/mediastream/gstreamer/GStreamerVideoCapturer.cpp

```diff
@@ -279,7 +279,7 @@ void GStreamerVideoCapturer::reconfigure()
 
     struct MimeTypeSelector {
         const char* mimeType = "video/x-raw";
-        const char* format = nullptr;
+        std::optional<ASCIILiteral> format;
```

Comment:
I think this should become a `<String>`

Reply:
Why?

This made me realize we call `gst_caps_set_simple()` with an `optional<ASCIILiteral>` that should be fixed :)

Reply:
It should become a `<String>` because you're keeping the format in the struct and you don't want to mess with the life time of the information you have gotten from the structure and also, conceptually, it is not a literal.